articles/libraries/lock-android/v1/index.md

```diff
@@ -0,0 +1,274 @@
+---
+section: libraries
+toc: true
+toc_title: Lock for Android v1
+title: Lock for Android v1
+url: /libraries/lock-android/v1
+description: A widget that provides a frictionless login and signup experience for your native Android apps.
+mobileimg: media/articles/libraries/lock-android.png
+---
+# Lock for Android
+
+<%= include('../_includes/_lock-version') %>
+
+## Requirements
+
+Android API level 15+ is required in order to use Lock's UI.
+If you'll create your own API and just call Auth0 API via the `com.auth0.android:core:1.13.+`, the minimum required API level is 9.
```

Comment:
Any mention of any library version (on every article in this PR) should point to the latest `1.18.+` or better, use `1.+` so it takes the latest patch version for v1. Check on https://search.maven.org if you need a clue.